### ISSUE TITLE
rotate option not working on Pi4 with Buster

### ISSUE THREAD

**Issue by eduardm** [popcornmix/omxplayer#765]
Running Raspbian Buster on a PI4b.

`omxplayer movi_file --orientation 90 ` produce non watchable video, see attached image

![IMG_1594](https://user-images.githubusercontent.com/33600/75604642-88f42880-5ae3-11ea-9723-e2e31020542e.jpg)

I tested with hundreds of videos, all working on older Pi / Raspbian versions.

**Comment by rafhtl** [comment]
it has been solved with 4.19.114

**Comment by popcornmix** [comment]
@eduardm is this resolved for you now?

**Comment by rafhtl** [comment]
Yes! I had the exact problem OMXPLAYER rotated 90 degrees and snow on the video
and now with rpi-update to 4.19.114  has solved
thank you
